product_pricelist_supplierinfo/readme/DESCRIPTION.rst

```diff
@@ -1,3 +1,5 @@
 This module allows you to create a sales pricelist based on product
 supplierinfo prices. If you want, you can bypass minimum quantity in pricelist
 item.
+We can also define sale marging applied on purchase price directly on supplier info.
```

Comment:
An empty line in between is needed here for proper RST syntax

Reply:
fixed, thanks for the catch